rest/oidc_api.go

```diff
@@ -176,7 +175,8 @@ func (h *handler) handleOIDCCallback() error {
 	}
 
 	// Converts the authorization code into a token.
-	token, err := client.Config.Exchange(context.Background(), code)
+	context := auth.GetClientContext(h.db.Options.UnsupportedOptions.OidcTlsSkipVerify)
```

Comment:
It seems we're setting a provider specific flag when we create the providers, but then still using the global SG config boolean for these.

Would it make sense to use the flag we set on the provider? I think it would allow for easier testing of the flag per-provider, as it's decoupled from the SG config.

```suggestion
	context := auth.GetClientContext(provider.InsecureSkipVerify)
```

Reply:
Yeah, it makes more sense when using the flag set over the provider. This is fixed in latest revision.